salt/base/sanity.sls

```diff
@@ -5,7 +5,18 @@ niceties:
       - htop
       - traceroute
 
-time-sync:
+{% if grains["oscodename"] in ["noble"] %}
+time-sync-timesyncd:
+    pkg.installed:
+      - pkgs:
+        - systemd-timesyncd
+
+systemd-timesyncd:
```

Comment:
This and the above block could be simplified a little to collapse it all into one top level key ala:

```
systemd-timesyncd:
  pkg:
    - installed
  service:
    - enabled: True
    - running
```

Reply:
Note: this kinda stuff only works in situations where the names all line up, in this case the service name and package name are identical. To see this taken to an extreme, our nginx state uses it to great effect. https://github.com/python/psf-salt/blob/f7803fd0142862bb3289b5944c598801f290fefd/salt/nginx/init.sls#L8-L53

Reply:
[ad57881](https://github.com/python/psf-salt/pull/386/commits/ad5788177e30015fd0326fe98f451cd8139b2af0) looks great!